ISICbalance/Services/RequestManager.swift

```diff
@@ -56,7 +56,7 @@ final class RequestManager: RequestManagering {
                     let lineElements = lineText.split(separator: " ")
                     guard lineElements.count > 0 else { throw RequestError.parseError(message: "Error - currency is missing") }
                     let balance = lineElements[0]
-                    let balanceStruct = Balance(balance: "\(balance) Kč")
+                    let balanceStruct = Balance(balance: String(balance))
```

Comment:
❔ `Kč` už se nebude zobrazovat?

Reply:
Kč se zobrazuje samostatně. Je tam přidělaná lokalizace a buď se to zobrazuje jako Kč nebo CZK.   
Ty nové změny jsou z namergovaného na 85% opraveného codereview od Honzy. Mergnul jsem si to do této branche a omylem odklikl push, takže v tom bude teď zmatek. Nečekal jsem, že na to někdo bude tak rychle reagovat. Opravím ty diskuze tedy v tom codereview a pak sem hodím zase merge.

Reply:
Systemový `NumberFormatter` umí pracovat s currency, pokud budeš mít `balance` jako číslo, tak ho určitě použij na formátování ceny